source/includes/Forms/Forms.md

```diff
@@ -0,0 +1,221 @@
+# Forms
+
+## Forms Overview
+
+* The concept dictionary is a central part of OpenMRS. Concepts can commonly be thought of as questions and possible answers which are on forms.
+* so forms in OpenMRS can be thought of as data collecting for various concepts in this dictionary corresponding to the questions these concepts need answers to. 
+* Forms in OpenMRS can be of many types, e.g., observation forms which help capture observations during an encounter e.t.c
+
+
+## Subresource types of Forms
+
+### formFields
+
+* The FormField object relates/orders the fields on a form. A form can have many 0 to n fields associated with it in a hierarchical manner. This FormField object governs what/how that takes place.
+
+
+## Available operations for Forms
+
+1.  [List Forms](#search-forms)
+3.  [Create a form](#create-a-form) 
+4.  [Update a form](#update-a-form)
+5.  [Delete a form](#delete-a-form)
+6.  [List formfields](#list-formfields)
+7.  [Create a formField subresource with properties](#create-a-formfield-subresource-with-properties)
+8.  [Update formFields subresource with properties](#update-formfields-subresource-with-properties)
+9.  [Delete formFields subresource with properties](#delete-formfields-subresource-with-properties)
+
+
+## List Forms
+
+* ### search-forms
+
+    Fetch all non-retired Forms that match any specified parameters otherwise fetch all non-retired forms. Returns a `200 OK` status with the form response,
+    and returns a `401` response when the user is not logged in. 
+
+    ### Query Parameters
+
+    Parameter | Description
+    --- | ---
+    *limit* | use this parameter to limit the number of results to be returned 
+    *startIndex* | the offset where to start the query
+    *v* | the required representation to return (i.e., ref, default, full or custom )
+    *q* | the search query
+    
+    
+```console
+GET /form?
+```
+
+* ### List forms by UUID
+
+    Retrieve a form by its UUID. Returns a `404 Not Found` status if the form does not exist in the system. If the user is not logged in to perform this action, a `401 Unauthorized` status is returned.
+
+    ### Query Parameters
+
+    Parameter | Description
+    --- | ---
+    *v* | the required representation to return (i.e. ref, default, full or custom )
+    *uuid* | the target form UUID 
+    
+```console
+GET /form/:target_form_uuid
+```
+
+## Create-a-form
+
+* To create a Form, you need to specify the below properties in the request. If you are not logged in to perform this action, a `401 Unauthorized` status is returned.
+
+    ### Properties
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *resource* | `form` | form resource with updated properties
```

Comment:
@Akayeshmantha  do we need to describe the properties of the resource here rather than just mentioning that it is a form ??

Reply:
of course we need that way. I don't see a point of keeping docs like this

Reply:
I will make the required changes @Akayeshmantha .